This file contains patches to make the DEC Ada 3.2 and 3.3 run-time library
compatible with GNAT and Ada95.

	* lib.ads: Specify F_FLOAT on CVTF_{TO,FROM}_INTERNAL_TIME.

$ gnudiff -c lib.ads.1 lib.ads
*** lib.ads	Fri Aug 29 06:15:19 1997
--- lib.ads	Fri Aug 29 06:16:13 1997
***************
*** 1413,1425 ****
      procedure CVTF_FROM_INTERNAL_TIME (
  	STATUS		: out COND_VALUE_TYPE;				-- return value
  	OPERATION	: in  FUNCTION_CODE_TYPE;
! 	RESULTANT_TIME	: out FLOAT;
  	INPUT_TIME	: in  DATE_TIME_TYPE);
   
      pragma INTERFACE (EXTERNAL, CVTF_FROM_INTERNAL_TIME);
   
      pragma IMPORT_VALUED_PROCEDURE (CVTF_FROM_INTERNAL_TIME, "lib$cvtf_from_internal_time",
! 	(COND_VALUE_TYPE, FUNCTION_CODE_TYPE, FLOAT, DATE_TIME_TYPE),
  	(VALUE, REFERENCE, REFERENCE, REFERENCE));
   
  --    LIB$CVT_FROM_INTERNAL_TIME
--- 1413,1425 ----
      procedure CVTF_FROM_INTERNAL_TIME (
  	STATUS		: out COND_VALUE_TYPE;				-- return value
  	OPERATION	: in  FUNCTION_CODE_TYPE;
! 	RESULTANT_TIME	: out F_FLOAT;
  	INPUT_TIME	: in  DATE_TIME_TYPE);
   
      pragma INTERFACE (EXTERNAL, CVTF_FROM_INTERNAL_TIME);
   
      pragma IMPORT_VALUED_PROCEDURE (CVTF_FROM_INTERNAL_TIME, "lib$cvtf_from_internal_time",
! 	(COND_VALUE_TYPE, FUNCTION_CODE_TYPE, F_FLOAT, DATE_TIME_TYPE),
  	(VALUE, REFERENCE, REFERENCE, REFERENCE));
   
  --    LIB$CVT_FROM_INTERNAL_TIME
***************
*** 1455,1467 ****
      procedure CVTF_TO_INTERNAL_TIME (
  	STATUS		: out COND_VALUE_TYPE;				-- return value
  	OPERATION	: in  FUNCTION_CODE_TYPE;
! 	INPUT_TIME	: in  FLOAT;
  	RESULTANT_TIME	: out DATE_TIME_TYPE);
   
      pragma INTERFACE (EXTERNAL, CVTF_TO_INTERNAL_TIME);
   
      pragma IMPORT_VALUED_PROCEDURE (CVTF_TO_INTERNAL_TIME, "lib$cvtf_to_internal_time",
! 	(COND_VALUE_TYPE, FUNCTION_CODE_TYPE, FLOAT, DATE_TIME_TYPE),
  	(VALUE, REFERENCE, REFERENCE, REFERENCE));
   
  --    LIB$CVT_TO_INTERNAL_TIME
--- 1455,1467 ----
      procedure CVTF_TO_INTERNAL_TIME (
  	STATUS		: out COND_VALUE_TYPE;				-- return value
  	OPERATION	: in  FUNCTION_CODE_TYPE;
! 	INPUT_TIME	: in  F_FLOAT;
  	RESULTANT_TIME	: out DATE_TIME_TYPE);
   
      pragma INTERFACE (EXTERNAL, CVTF_TO_INTERNAL_TIME);
   
      pragma IMPORT_VALUED_PROCEDURE (CVTF_TO_INTERNAL_TIME, "lib$cvtf_to_internal_time",
! 	(COND_VALUE_TYPE, FUNCTION_CODE_TYPE, F_FLOAT, DATE_TIME_TYPE),
  	(VALUE, REFERENCE, REFERENCE, REFERENCE));
   
  --    LIB$CVT_TO_INTERNAL_TIME
